app/src/main/java/com/cornellappdev/android/eatery/data/CafeteriaDbHelper.java

```diff
@@ -8,96 +8,100 @@
-  public void onCreate(SQLiteDatabase db) {
-    final String SQL_CREATE_TABLE =
-        "CREATE TABLE "
-            + CafeteriaContract.CafeteriaEntry.TABLE_NAME
-            + " ("
-            + CafeteriaContract.CafeteriaEntry._ID
-            + " INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL, "
-            + CafeteriaContract.CafeteriaEntry.COLUMN_DATA
-            + " TEXT NOT NULL"
-            + ");";
-    db.execSQL(SQL_CREATE_TABLE);
-  }
+    @Override
+    public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
+        db.execSQL("DROP TABLE IF EXISTS " + CafeteriaContract.CafeteriaEntry.TABLE_NAME);
+        onCreate(db);
+    }
 
-  @Override
-  public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
-    db.execSQL("DROP TABLE IF EXISTS " + CafeteriaContract.CafeteriaEntry.TABLE_NAME);
-    onCreate(db);
-  }
+    public boolean addData(String json) {
+        ContentValues values = new ContentValues();
+        values.put(CafeteriaContract.CafeteriaEntry.COLUMN_DATA, json);
+        SQLiteDatabase db = getWritableDatabase();
+        long result = db.insert(CafeteriaContract.CafeteriaEntry.TABLE_NAME, null, values);
+        db.close();
+        return result == -1 ? false : true;
+    }
 
-  public boolean addData(String json) {
-    ContentValues values = new ContentValues();
-    values.put(CafeteriaContract.CafeteriaEntry.COLUMN_DATA, json);
-    SQLiteDatabase db = getWritableDatabase();
-    long result = db.insert(CafeteriaContract.CafeteriaEntry.TABLE_NAME, null, values);
-    db.close();
-    return result == -1 ? false : true;
-  }
+    public String getLastRow() {
+        String dbString = "";
+        SQLiteDatabase db = getWritableDatabase();
+        String query =
+                "SELECT * FROM "
+                        + CafeteriaContract.CafeteriaEntry.TABLE_NAME
+                        + " ORDER BY "
+                        + CafeteriaContract.CafeteriaEntry._ID
+                        + " DESC LIMIT 1";
+        Cursor cursor = db.rawQuery(query, null);
+        if (cursor.moveToFirst()) {
+            dbString =
+                    cursor.getString(
+                            cursor.getColumnIndex(CafeteriaContract.CafeteriaEntry.COLUMN_DATA));
+        }
+        cursor.close();
+        return dbString;
+    }
 
-  public String getLastRow() {
-    String dbString = "";
-    SQLiteDatabase db = getWritableDatabase();
-    String query =
-        "SELECT * FROM "
-            + CafeteriaContract.CafeteriaEntry.TABLE_NAME
-            + " ORDER BY "
-            + CafeteriaContract.CafeteriaEntry._ID
-            + " DESC LIMIT 1";
-    Cursor cursor = db.rawQuery(query, null);
-    if (cursor.moveToFirst()) {
-      dbString =
-          cursor.getString(cursor.getColumnIndex(CafeteriaContract.CafeteriaEntry.COLUMN_DATA));
+    public void removeData() {
+        SQLiteDatabase db = getWritableDatabase();
+        db.execSQL(
+                "DELETE FROM "
```

Comment:
prefer a format string here instead, so define it like this:

```
String deleteQuery = "DELETE FROM %s WHERE %s=1"
db.execSQL(deleteQuery.format(..., ...))
```

The syntax might be off, but the concept should be clear